integration-tests/src/test/java/com/amazonaws/services/schemaregistry/integrationtests/mm2converter/GlueSchemaRegistryConverterIntegrationTest.java

```diff
@@ -0,0 +1,251 @@
+        converter = new CrossRegionReplicationConverter();
+    }
+
+    @AfterAll
+    public static void tearDown() throws URISyntaxException {
+        Map<String, Object> properties = getConverterProperties();
+        GlueClient glueClientAtSourceRegion = GlueClient.builder()
+                .credentialsProvider(defaultCredProvider)
+                .region(Region.of((String) properties.get(AWSSchemaRegistryConstants.AWS_SOURCE_REGION)))
+                .endpointOverride(new URI((String) properties.get(AWSSchemaRegistryConstants.AWS_SOURCE_ENDPOINT)))
+                .httpClient(UrlConnectionHttpClient.builder()
+                        .build())
+                .build();
+
+        GlueClient glueClientAtTargetRegion = GlueClient.builder()
+                .credentialsProvider(defaultCredProvider)
+                .region(Region.of((String) properties.get(AWSSchemaRegistryConstants.AWS_TARGET_REGION)))
+                .endpointOverride(new URI((String) properties.get(AWSSchemaRegistryConstants.AWS_TARGET_ENDPOINT)))
+                .httpClient(UrlConnectionHttpClient.builder()
+                        .build())
+                .build();
+
+        for (String schemaName : schemasToCleanUp) {
+            DeleteSchemaRequest deleteSchemaRequest = DeleteSchemaRequest.builder()
+                    .schemaId(SchemaId.builder()
+                            .registryName("default-registry")
+                            .schemaName(schemaName)
+                            .build())
+                    .build();
+
+            glueClientAtSourceRegion.deleteSchema(deleteSchemaRequest);
+            glueClientAtTargetRegion.deleteSchema(deleteSchemaRequest);
+        }
+    }
+
+    /**
+     * Test when Avro schemas are replicated through MM2 successfully.
+     */
+    @ParameterizedTest
+    @MethodSource("schemaRecordPairs")
+    public void SchemaReplication_AvroData_Succeeds(Schema schema, GenericRecord record) {
+        KafkaProducer<String, GenericRecord> producer = new KafkaProducer<>(getKafkaProducerProperties());
+        ProducerRecord<String, GenericRecord> newRecord = new ProducerRecord<>(schema.getName(), record);
+        producer.send(newRecord);
+        schemasToCleanUp.add(schema.getName());
+
+        Properties consumerProperties = getKafkaConsumerProperties();
+        consumerProperties.replace(ConsumerConfig.GROUP_ID_CONFIG, UUID.randomUUID().toString());
+        KafkaConsumer<String, GenericRecord> consumer = new KafkaConsumer<>(consumerProperties);
+        consumer.subscribe(Collections.singletonList("A." + schema.getName()));
+        Schema returnedSchema = null;
+        ConsumerRecords<String, GenericRecord> returnRecords = consumer.poll(1000);
+        for (ConsumerRecord<String, GenericRecord> returnRecord : returnRecords) {
+            GenericRecord value = returnRecord.value();
+            returnedSchema = value.getSchema();
+        }
+        assertEquals(schema, returnedSchema);
```

Comment:
Should we assert that the schema version Ids are different? Any other attributes to assert like metadata (transport name)?